From 7b6e120d483e628f1f0ffd397e5b4f7230c967be Mon Sep 17 00:00:00 2001
From: Mihai Serban <mihai.serban@intel.com>
Date: Fri, 4 Sep 2015 17:50:21 +0300
Subject: [PATCH 10/11] iio: increase limit of trigger names to 64 characters

The current limit of 32 characters makes impossible usage of triggers with long
names like hrtimer-lsm303dlhc_accel-hr-dev3.

Change-Id: If600d9336736ee2da1035baa70198be531f0b3a3
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-26476
Signed-off-by: Mihai Serban <mihai.serban@intel.com>
Reviewed-on: https://android.intel.com:443/486303
---
 drivers/iio/industrialio-configfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/iio/industrialio-configfs.c b/drivers/iio/industrialio-configfs.c
index daf318c..3574221 100644
--- a/drivers/iio/industrialio-configfs.c
+++ b/drivers/iio/industrialio-configfs.c
@@ -17,7 +17,7 @@
 #include <linux/iio/iio.h>
 #include <linux/iio/sw_trigger.h>
 
-#define MAX_NAME_LEN 32
+#define MAX_NAME_LEN 64
 
 static struct config_group *trigger_make_group(struct config_group *group,
 					       const char *name)
-- 
1.9.1

